modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -201,6 +210,13 @@ internal class DiscoverAdapter(
             snapHelper.attachToRecyclerView(recyclerView)
             snapHelper.onSnapPositionChanged = { position ->
                 binding.pageIndicatorView.position = position
+                val row = getItem(position) as? DiscoverRow
+                row?.listUuid?.let {
+                    analyticsTracker.track(
```

Comment:
Trending and popular lists might not have a `listUuid` as I don't see them as nested lists in the admin area. 
I realized a few problems here:

- It should have been best to use the inferred id to track this event as the lists don't have `listUuid` set on them.

- I was doing wrong by passing `position` instead of the `bindingAdapterPosition` to get the `DiscoverRow` id. Here, `position` corresponds to the paginator position and `bindingAdapterPosition` is the position of the `DiscoverRow` w.r.t. the main vertical list. I fixed it in d8841e9. 

I verified that the events were fired correctly now, also matched with iOS:

```
🔵 Tracked: discover_small_list_page_changed, Properties: {"current_page":1,"total_pages":5,"list_id":"trending"
🔵 Tracked: discover_small_list_page_changed, Properties: {"current_page":1,"total_pages":5,"list_id":"popular"
```